src/features/projectsV2/ProjectsMap/MapFeatureExplorer/microComponents/MapLayerControlPanel.tsx

```diff
@@ -0,0 +1,69 @@
+import type { ChangeEvent } from 'react';
+import type { MapOptions } from '../../../ProjectsMapContext';
+
+import { StyledSwitch } from '../CustomSwitch';
+import styles from '../MapFeatureExplorer.module.scss';
+import LayerSwitchContainer from './LayerSwitchContainer';
+
+export type AdditionalInfo = {
+  dataYears: number;
+  resolution: string;
+  description: string;
+  underlyingData: string;
+  source: string;
+  covariates: string;
+};
+
+interface Props {
+  category?: string;
+  exploreConfig: {
+    label: string;
+    color: string | undefined;
+    showDivider: boolean;
+    additionalInfo?: AdditionalInfo;
+    shouldRender: boolean;
+  }[];
+  mapOptions?: MapOptions;
+  updateMapOption?: (option: keyof MapOptions, value: boolean) => void;
+}
+
+const MapLayerControlPanel = ({
+  category,
```

Comment:
I feel it may be more appropriate to name this `MapSettingsSection` as it conveys the purpose of this component better (grouping).